rosdep/python.yaml

```diff
@@ -1449,6 +1449,10 @@ python-scipy:
     utopic_python3: [python3-scipy]
     vivid: [python-scipy]
     vivid_python3: [python3-scipy]
+python-selenium:
+  ubuntu:
+    pip:
+      packages: [selenium]
```

Comment:
Why not rely on the Debian package available on Ubuntu: http://packages.ubuntu.com/trusty/python/python-selenium